Tests/scripts/configure_tests.py

```diff
@@ -47,6 +48,8 @@
 # secrets white list file to be ignored in tests to prevent full tests running each time it is updated
 SECRETS_WHITE_LIST = 'secrets_white_list.json'
 
+# pack_metadata.json is the local metadata for a pack
```

Comment:
what is this comment relates to?

Reply:
to nothing, it's a leftover. I'll remove it